src/aws-cur/README.md

```diff
@@ -23,7 +23,7 @@ Here's how you can create a monthly cost and usage report in Parquet format:
 
 ```typescript
 const app = new App();
-const stack = new Stack(app, 'CostReportStack');
+const stack = new Stack(app, 'CostReportStack', { env: { region: 'us-east-1' } });
```

Comment:
Can you please state that regions other than `us-east-1` cannot be specified?

Reply:
Thank you for reviewing!
I have added comment.